src/pages/StepStart/StepStart.tsx

```diff
@@ -42,27 +43,19 @@ export function StepStart() {
     >
       <Await
         resolve={Promise.all([
-          loaderData.getPersonQuery,
-          loaderData.getLoependeVedtakQuery,
-          loaderData.shouldRedirectTo,
+          getPersonQuery,
+          getLoependeVedtakQuery,
+          shouldRedirectTo,
         ])}
       >
-        {(queries: [GetPersonQuery, GetLoependeVedtakQuery, string]) => {
-          const getPersonQuery = queries[0]
-          const getVedtakQuery = queries[1]
-          const shouldRedirectTo = queries[2]
-
+        {(resp: [GetPersonQuery, GetLoependeVedtakQuery, string]) => {
```

Comment:
Hadde kanskje vært lettere å destrukturere listen her, slik man slipper resp[0]

eks. {[personResponse, vedtakResponse, redirectToResponse]: [GetPersonQuery, GetLoependeVedtakQuery, string]}

i stedet for {(resp: [GetPersonQuery, GetLoependeVedtakQuery, string])